modules/Presentation/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
1) The gitignore does not belong here and should be removed. 
2) The folder structure does not match the rest of the project. I think it should be modules/Presentation.py and not in another folder.